stories/ui/RecommendedBadge.js

```diff
@@ -0,0 +1,15 @@
+/* @flow */
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import RecommendedBadge from 'ui/components/RecommendedBadge';
+
+import Provider from '../setup/Provider';
+
+storiesOf('RecommendedBadge', module)
+  .addDecorator((story) => (
+    <div className="RecommendedBadge--storybook">
+      <Provider story={story()} />
+    </div>
+  ))
+  .add('default', () => <RecommendedBadge />);
```

Comment:
I don't know if that's feasible but a RTL story would be nice.

Reply:
We can do that! https://www.npmjs.com/package/storybook-addon-rtl